torch/_tensor_docs.py

```diff
@@ -1714,7 +1714,7 @@ def callable(a, b) -> number
 
 add_docstr_all('expand',
                """
-expand(tensor, sizes) -> Tensor
+expand(*sizes) -> Tensor
```

Comment:
Actually, `expand` accept both `*sizes` as well as `torch.Size`, so this is incomplete.